.fixtures.yml

```diff
@@ -1,6 +1,8 @@
 fixtures:
   repositories:
+    apt: "https://github.com/puppetlabs/puppetlabs-apt"
```

Comment:
are the changes in the .fixtures.yml actually required? the exporter doesn't use apt/nginx, only the acceptance tests. but the fixtures.yml is only used within unit tests. a possible way to install modules in acceptance tests is this: https://github.com/voxpupuli/puppet-grafana/blob/master/spec/acceptance/class_spec.rb#L8-L10

Reply:
ah, didn't know this.

the modules are installed in `spec/acceptance/nginx_prometheus_exporter_spec.rb` with:

```
describe 'prometheus nginx exporter' do
  it 'nginx_prometheus_exporter works idempotently with no errors' do
    on hosts, puppet('module', 'install', 'puppet-nginx', '--version', '3.2.1')
    on hosts, puppet('module', 'install', 'puppetlabs-apt', '--version', '8.2.0')
```